documentation/api/io.strimzi.api.kafka.model.CruiseControlSpec.adoc

```diff
@@ -231,6 +231,16 @@ spec:
         outboundNetwork: 30000KiB/s
 ----
 
+If you are using `cruiseControl.brokerCapacity.cpu` or `cruiseControl.brokerCapacity.overrides.cpu` to change the default capacity limits for goals related to resource distribution, the order of precedence is as follows, with highest priority first:
```

Comment:
The `If you are using` makes it sound that this applies only when those two fields are used. But the order is valid always. So I would maybe try to rephrase it?

```suggestion
The order of precedence is as follows, with highest priority first:
```